.github/workflows/documentation.yml

```diff
@@ -48,16 +48,17 @@ jobs:
     - name: Install Dependencies 📦
       run: make install
 
-    - name: Build & Publish Docs 🏃‍♀️
-      env:
-        GITHUB_DOCS_KEY: ${{ secrets.GITHUB_DOCS_KEY }}
-      run: |
-        eval "$(ssh-agent -s)"; touch docs_key; chmod 0600 docs_key
-        echo "$GITHUB_DOCS_KEY" > "docs_key"
-        ssh-add docs_key
+    # FIXME: https://github.com/RasaHQ/rasa/issues/6166
```

Comment:
I think it'd be wise to have the code ready for this issue BEFORE we merge the initial PR. Otherwise we could leave `master` with no easy way to release docs. And it could be detrimental to alpha2.

Reply:
(getting started today)

Reply:
yes that sounds good, better to have some sort of deployment of the docs that we can share.